endpoints_management/control/wsgi.py

```diff
@@ -240,8 +240,8 @@ def __init__(self,
            """
         self._application = application
         self._project_id = project_id
-        self._next_operation_id = next_operation_id
         self._control_client = control_client
+        self._next_operation_id = next_operation_id
```

Comment:
Any reason for this change?

Reply:
I reordered the assignments to match the argument order.